wsd/DocumentBroker.cpp

```diff
@@ -196,6 +196,7 @@ DocumentBroker::DocumentBroker(ChildType type, const std::string& uri, const Poc
     , _configId(configId)
     , _poll(
           std::make_shared<DocumentBrokerPoll>("doc" SHARED_DOC_THREADNAME_SUFFIX + _docId, *this))
+    , _slideLayerCache(25)
```

Comment:
If the max size of the cache will be quite small, like a hundred or so it might be better to not bother with a separate std::map of results and a std::vector of their order and instead just have a simple std::vector of results and use the ordering of that as the order and iterate through it to find the result on lookup. Probably don't really get much benefit of the quicker std::map lookup unless the number of entries is significant. But lets see what this number ends up as before worrying about that I guess.